src/_data/catalog/widgets.json

```diff
@@ -11,13 +11,15 @@
   },
   {
     "name": "AlertDialog",
-    "description": "Alerts are urgent interruptions requiring acknowledgement that inform the user about a situation. The AlertDialog widget implements this component.",
+    "description": "Hovering containers that prompt app users to provide more data or make a decision.",
```

Comment:
This updates AlertDialog to contain the info added to "Dialog" in 167766d815566eaeddd6540d4dd46cfcfe6d8cca  - which deduplicates this